app/models/visit.rb

```diff
@@ -76,4 +76,8 @@ def banned_visitors
   def unlisted_visitors
     visitors.unlisted
   end
+
+  def unavailable?
```

Comment:
`awaiting_processing?` won't work either as it includes `booked` and `rejected`...I'll think about it and see if I can come up with something more suitable.